jetty-server/src/main/java/org/eclipse/jetty/server/session/SessionHandler.java

```diff
@@ -654,7 +655,7 @@ public String getSessionCookie()
      */
     public HttpCookie getSessionCookie(HttpSession session, String contextPath, boolean requestIsSecure)
     {
-        if (isUsingCookies())
+        if (isUsingCookies() && _cookieConfig.getName() != null)
```

Comment:
I don't think that testing _cookieConfig.getName() for null should be here. As per issue https://github.com/eclipse-ee4j/jakartaee-tck/pull/567 it looks like cookieConfig.getName() should indeed be able to return null, but the container needs to either use it's own default name, or the spec default name in that case.

We should also be consistent in SessionCookieConfig:  if you look at getPath(), we allow that to return null, but when we need to have a path, we substitute the context path of the relevant context.

Reply:
I've changed the PR so null is now acceptable and uses a static method to convert to the default